types/google-apps-script/google-apps-script.drive.d.ts

```diff
@@ -32,104 +32,29 @@ declare namespace GoogleAppsScript {
     export interface DriveApp {
       Access: typeof Access;
       Permission: typeof Permission;
-      /**
-       * Adds the given file to the root of the user's Drive.
-       * This method does not move the file out of its existing parent folder;
-       * a file can have more than one parent simultaneously.
-       */
```

Comment:
Don't remove these.

Reply:
Thanks for reviewing! I did not change anything in the files auto-generated by [dts-google-apps-script](https://github.com/motemen/dts-google-apps-script), so I will be manually replacing `String` with `string` and re-inserting comments, just to be clear.

Reply:
Also, I'm not sure why the Travis CI tests are failing or how to make them pass.